addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.js

```diff
@@ -26,6 +26,10 @@ class ActivityMarkDonePopover extends Component {
     // Public
     //--------------------------------------------------------------------------
 
+    mounted() {
+        this._feedbackTextareaRef.el.focus();
+    }
```

Comment:
Probably should have a `isDoFocus` and not rely just on `mounted` but also on `patched`.

Reply:
I find the `IsDoFocus` a little bit strange as the model is the activity and not the popover, so the text input is not really linked to the activity... Beside this, it's only executed on mounted normally as it's a Popover, so it's always focused on open (so mounted) and then it's destroyed. Don't you think ?

Reply:
Considering all the other issues still present with Popover in general, I would be ok to be less strict about this one.

Reply:
Not great, but would need to introduce viewer of some sort, so don't bother with it right now.